src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpHelper.h

```diff
@@ -73,13 +73,50 @@ PolyDriverDescriptor constructRemoteControlBoardRemapper(
+ * | `local_port_name_postfix` | `string` | Postfix of the local port. The local port name is `/<local_prefix><local_port_name_postfix>` |    Yes    |
  * @note The genericSensorClient device is implement in [whole-body-estimators](https://github.com/robotology/whole-body-estimators/tree/cc8ffb83375a2d410b225f4fa67aee4a29074b42/devices/genericSensorClient).
  * @return A PolyDriverDescriptor. If one of the parameters is missing an invalid PolyDriverDescriptor is returned.
  */
 PolyDriverDescriptor constructGenericSensorClient(
     std::weak_ptr<const BipedalLocomotion::ParametersHandler::IParametersHandler> handler);
 
+/**
+ * Helper function that can be used to build a `MultipleAnalogSensorsClient` device.
+ * @param handler pointer to a parameter handler interface.
+ * @note the following parameters are required by the function
+ * |       Parameter Name      |   Type   |                                          Description                                         | Mandatory |
+ * |:-------------------------:|:--------:|:--------------------------------------------------------------------------------------------:|:---------:|
+ * |       `description`       | `string` |      Description of the multiple analog sensor client. It is the device name                 |    Yes    |
+ * |     `remote_port_name`    | `string` |                           Name of the port associate to the remote                           |    Yes    |
+ * |       `local_prefix`      | `string` |                     Prefix of the local port (e.g. the application name)                     |    Yes    |
+ * | `local_port_name_postfix` | `string` | Postfix of the local port. The local port name is `/<local_prefix><local_port_name_postfix>` |    Yes    |
+ * |           `timeout`       | `double` |         Timeout in seconds after which the device reports an error if no measurement was received. (Default value 0.01)      |     No    |
+ * |   `external_connection`   |  `bool`  | If set to true, the connection to the rpc port of the MAS server is skipped and it is possible to connect to the data source externally after being opened. (Default value `false`)      |     No    |
+ * |   `carrier`    | `string` |  The carrier used for the connection with the server. (Default value `tcp`)      |     No    |
+ * @note The `MultipleAnalogSensorsClient` device is implement in [yarp](https://www.yarp.it/git-master/classMultipleAnalogSensorsClient.html).
+ * @return A `PolyDriverDescriptor`. In case of error an invalid `PolyDriverDescriptor` is returned.
+ */
+PolyDriverDescriptor constructMultipleAnalogSensorsClient(
+    std::weak_ptr<const BipedalLocomotion::ParametersHandler::IParametersHandler> handler);
+
+/**
+ * Helper function that can be used to build a `MultipleAnalogSensorsRemapper` device.
+ * @param handler pointer to a parameter handler interface.
+ * @note the following parameters are required by the function
```

Comment:
Nitpick. In the table below, all parameters are not mandatory, hence, they are not "required".

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/568/commits/07262780d701704dba11befbc7df08ac71e12393